platform/kube/controller.go

```diff
@@ -363,6 +363,26 @@ const (
 // "spiffe://cluster.local/ns/foo/sa/bar".
 func (c *Controller) GetIstioServiceAccounts(hostname string, ports []string) []string {
 	saSet := make(map[string]bool)
+
+	// Get the service accounts allowed for the service running on VMs, specified by the Service
```

Comment:
Why plural ( on K8S there is a single service account - why would we need multiple in VM ) ? 
And can we just specify the service account the same as in K8S ( using the normal yaml tag ) ?